c_emulator/riscv_platform.h

```diff
@@ -38,3 +38,6 @@ unit plat_term_write(mach_bits);
 mach_bits plat_htif_tohost(unit);
 
 unit memea(mach_bits, sail_int);
+
+// Zawrs : Wait-on-Reservation-Set
```

Comment:
I think we should be consistent and either always use C++ style // comments or always C-style